examples/gno.land/p/demo/uuid/uuid.gno

```diff
@@ -0,0 +1,115 @@
+// Copyright 2023 The godruoyi/gno-snowflake Author. All rights reserved.
```

Comment:
This is a modified version, so you need to adapt the license message to maintain full attribution while indicating that it is no longer the original file.